clang/test/Analysis/exercise-ps.c

```diff
@@ -22,7 +22,7 @@ void_typedef f2_helper(void);
 static void f2(void *buf) {
   F12_typedef* x;
   x = f2_helper();
-  memcpy((&x[1]), (buf), 1); // expected-warning{{call to undeclared library function 'memcpy' with type 'void *(void *, const void *}} \
+  memcpy((&x[1]), (buf), 1); // expected-warning{{call to undeclared library function 'memcpy', will assume it exists with standards-mandated type 'void *(void *, const void *,}} \
```

Comment:
Note, there's another type in the parameter list, but it's size_t which resolves to different types on different systems, so it's dropped here for the same reason it was dropped before.